File: AndroidStudioProjects/matrix/app/src/main/java/com/example/matrix/MainActivity.java
package com.example.matrix;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private EditText editTextNumber2;
    private Button button;
    private TextView textview2;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        editTextNumber2=findViewById(R.id.editTextNumber2);
        button=findViewById(R.id.button);
        textview2=findViewById(R.id.textView2);




        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                String a=editTextNumber2.getText().toString();
                int b= Integer.parseInt(a);
                 for(int i=1;i<10;i++) {
                     int j = i * b;
                     textview2.setText(i + " * " + b + " is " + j + "\n");


                 }

            }
        });

    }
}

File: AndroidStudioProjects/ApplicationForm/app/src/main/java/com/example/applicationform/MainActivity.kt
package com.example.applicationform


import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import android.widget.Toast


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val nameTextBox=findViewById<TextView>(R.id.nameTextBox)
        val mySendButton=findViewById<Button>(R.id.mySendButton)
      var name = " "
        name =nameTextBox.text.toString()

        mySendButton.setOnClickListener{
            name=nameTextBox.text.toString()
            Toast.makeText(applicationContext, name, Toast.LENGTH_SHORT).show()
            val intent=Intent(this,MainActivity2::class.java).apply {
                putExtra("Refer-name",name)
            }
            startActivity(intent)
        }



    }
}





